cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
I've been able to duplicate the reboot issue with `examples/ipv6/simple-udp-rpl` and `noncoresec` in the `master` branch. The watchdog timer is triggered by a deadlock caused by `aes_auth_crypt_check_status()` never indicating an end of operation. I think that the root cause is that the status flags are cleared after the final register write access starting the operation, so they can be cleared after having been set, depending on the timing. I will investigate this by identifying the precise register write access starting each operation, thanks to the UG and some tests.